code/style.css

```diff
@@ -1,58 +1,199 @@
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: 'Montserrat', sans-serif;
+  background: rgb(31, 38, 74);
+  background: radial-gradient(circle, rgba(31, 38, 74, 1) 0%, rgba(32, 25, 27, 1) 100%);
 }
 
 p {
   margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+  font-size: 18px;
+  color: white;
 }
 
+h1 {
+  font-family: 'Montserrat', sans-serif;
+}
+
+/* LOGO CONTAINER */
+
+.top-container {
+  display: flex;
+  justify-content: center;
+  margin: 40px;
+}
+
+.top-container img {
+  width: 500px;
+  border-radius: 5px;
+}
+
+/* MYSTERY CARDS */
+
 .cards {
-  margin: 50px auto;
+  margin: 100px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
   display: flex;
-  justify-content: space-around;
+  justify-content: space-evenly;
   align-items: center;
+  z-index: 1;
 }
 
-.killer-deck {
+.killer-deck,
+.weapon-deck,
+.room-deck {
   width: 250px;
-  height: 400px;
-  background: #000;
+  height: 300px;
   text-align: center;
-  padding-top: 30px;
   border-radius: 16px;
+  position: relative;
+  overflow: hidden;
+  transition: .3s ease-in-out;
 }
 
-.weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+.killer-deck p,
+.weapon-deck p,
+.room-deck p {
```

Comment:
Since these are all the same, maybe they could have a more generic class name? Like, card-decks p. The you would not have to add so much code